### ISSUE TITLE
Title: Enhancing the overall UI of the website's page

### ISSUE THREAD

**Issue by 07tAnYa** [swapnilsparsh/30DaysOfJavaScript#1173]
### Which pre-existing JavaScript project will you be working on?

I would be working on the overall look of the page, which will include all the JS projects.

### Describe your enhancement/improvement

I would try to fix the sizing of the icons, the spacing and yes the color-combo too.

### Anything you need to say to the Reviewers

Kindly review this issue, as I think this will contribute to the overall enhanced ui for the project.
And if this finds genuine to you, then kindly assign this to me.

### Add any other context or relevant screenshots about the enhancement here.

![Screenshot 2024-05-10 164714](https://github.com/swapnilsparsh/30DaysOfJavaScript/assets/140234197/e1209b2f-1bca-42d1-b500-875928d460fd)

**Comment by 07tAnYa** [comment]
Please assign this issue to me if approved. Thank you.

**Comment by Saloni-0465** [comment]
I've been exploring the 30 Days of JavaScript website, and I've noticed several areas where the user interface could be enhanced. Specifically, I've identified the following issues:

Clickable boards: I noticed that the mini projects displayed in boards are not clickable directly from the board itself. Instead, users have to click on the link icon at the bottom of the board to access the project

Button Styling: The buttons on the site could benefit from improved styling to make them more visually appealing and intuitive to use.

Search Bar Design: The design of the search bar could be improved to make it more prominent and aesthetically pleasing.

Font and Color Theme: The font choices and color theme throughout the site could be reconsidered to create a more cohesive and visually appealing design.
